tools/deploy-with-olm.sh

```diff
@@ -47,24 +58,42 @@ exit_on_err make docker-push IMG=${DEPLOYER_IMAGE}
 # Generate the olm bundle image
 blue "Generate and push the olm bundle image: ${BUNDLE_IMAGE}"
 exit_on_err make bundle IMG=${DEPLOYER_IMAGE}
+exit_on_err sed -i "s|sed_replace_addon_name|${ADDON_NAME}|" ${BUNDLE_FILE}
+exit_on_err sed -i "s|sed_replace_sop_endpoint|${SOP_ENDPOINT}|" ${BUNDLE_FILE}
```

Comment:
Can we use `yq`, which actually understands the YAML format,  instead of `sed`. 
The benefit will also be that we will not need to introduce keys for the find/replace logic

Reply:
I checked the documentation for `yq`, but couldn't find any functionality that does field replaement as is being done here. Do you have any examples of this?

Reply:
See the following SO answer https://stackoverflow.com/questions/66446495/how-to-patch-multi-document-yaml-file-on-condition-using-yq

You can do it via the `|=` op or use kustomize which we already have in our repo